src/main/java/io/confluent/connect/jdbc/sink/JdbcSinkConfig.java

```diff
@@ -249,8 +249,9 @@
   private static final String TABLE_TYPES_DOC =
       "The comma-separated types of database tables to which the sink connector can write. "
       + "By default this is ``" + TableType.TABLE + "``, but any combination of ``"
-      + TableType.TABLE + "`` and ``" + TableType.VIEW + "`` is allowed. Not all databases "
-      + "support writing to views, and when they do the the sink connector will fail if the "
+      + TableType.TABLE + "``, ``" + TableType.PARTITIONED_TABLE + "`` and ``"
+      + TableType.VIEW + "`` is allowed. Not all databases support writing to views, "
```

Comment:
Ideally I'd like to support partitioned tables by default as well, although I didn't want to just add that without checking here first. Say the word and I will add it to the MR.

Reply:
I think at this point, default value of Table should be ok. Unless you think supporting partitioned tables by default is important?

Reply:
For functionality that used to just work out of the box I think it's kindest to restore that here by default.
Otherwise at the very least we would have to document how configure this for users, which would be more work.

Reply:
I think it would be better to no touch the current default and document how to configure partitioned table support.

Reply:
I wouldn't know how to configure this myself; could you help guide me in updating the documentation?